python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -116,7 +116,7 @@ def broadcast(*columns: Column, target_length: int | None = None) -> list[Column
             )
     return [
         column
-        if column.size != 1
+        if column.obj.size() != 1
```

Comment:
Why this change?

Reply:
Some of these changes are accidental from working off an old branch. I'll be sure to clean this up (thanks for pointing it out!)